### DISCUSSION TITLE
Can I help with the documentation?

### DISCUSSION THREAD

**Discussion by Fatih20** [xournalpp/xournalpp#3283] [Ideas]
The website seems to mention how inadequate the current user manual is. I want to contribute to it.

**Comment by rolandlo** [comment]
I think the current user manual should be removed and replaced by a link to the [website](https://xournalpp.github.io). Only technical stuff that we don't want the average user to bother with should be in the [Wiki](https://github.com/xournalpp/xournalpp/wiki). Everything accessible to the average user should be on the website.
